inst/odin/carehomes.R

```diff
@@ -42,7 +44,9 @@ n_I_P_vaccinated[, ] <-
   sum(n_II_P_next_vacc_class[i, , , j])
 dim(n_I_P_vaccinated) <- c(n_groups, n_vacc_classes)
 n_R_vaccinated[, ] <-
-  sum(n_R_next_vacc_class[i, , j]) + sum(n_RS_next_vacc_class[i, , j])
+  sum(n_RR_next_vacc_class[i, , j]) +
```

Comment:
I would perhaps rename n_RR_next_vacc_class to n_R_next_vacc_class as they move vaccination class but they don't move from R to the next R compartment which the current name suggests I think.